### DISCUSSION TITLE
Running a workflow until other workflows finish

### DISCUSSION THREAD

**Discussion by sbanoseg** [community/community#33738] [Actions]
Hello team, actually I have several workflows configured in my repo, but I was wondering if there is a way to trigger one of those workflows after the other ones have finished, I mean, let's suppose I have 5 workflows named (A,B,C,D,E), but I would like to execute the workflow C until the other four workflows have finished.

**Comment by sbanoseg** [comment]
Any help would be appreciated

**Comment by Simon-Ravitz** [comment]
Hey, @sbanoseg
There's a kind of workaround by triggering each workflow with a different event
But need to keep in mind that need to create a logical order of the workflow executions.
In addition, it's worth checking probably the order of the workflow files is by alphabet, so you can try to rename the files as a,b,c..
Thanks
